WultraMobileTokenSDK/Operations/Model/UserOperation/WMTUserOperation.swift

```diff
@@ -61,4 +61,9 @@ open class WMTUserOperation: WMTOperation, Codable {
     
     /// Proximity Check Data to be passed when OTP is handed to the app
     public var proximityCheck: WMTProximityCheck?
+    
+    /// Optional details why the status has changed by backend services.
```

Comment:
I would change the wording here (mainly because only the first sentence is read, not the description):

```
Enum-like reason why the status has changed.

Max 32 characters are expected. Possible values depend on the backend implementation and configuration.
```